packages/itwinui-react/scripts/build.mjs

```diff
@@ -25,13 +25,21 @@ const swcOptions = {
   ].join(' -C '),
 
   get esm() {
-    return [this.shared, this.compilerOptions, '-C module.type=es6'].join(' ');
+    return [
+      this.shared,
+      this.compilerOptions,
+      '-C module.type=es6',
+      `-C jsc.transform.optimizer.globals.vars.__module="\'ESM\'"`,
```

Comment:
Hmm. For most dev servers we use the TypeScript files directly, so it would be complicated to make it work.

I'll see if I can work around the error. The actual value is not important for dev servers at all.